docs/basics/command-line.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "Command-line"
+description: "A simple walkthrough of how to perform basic IPFS operations using the command-line."
+---
+
+# Basic CLI Operations
+
+This short guide aims to walk you through the basics of using IPFS with the CLI (Command Line Interface). You will learn how to add, retrieve, read, and remove files from within the CLI. We will try to keep it simple here, if you are unsure about the meaning of some terms you can check out the [glossary](../concepts/glossary/). Also don't forget to check out the rest of the [docs](../) to learn more about IPFS!
+
+All instructions and examples shown here were performed and tested on an M1 Mac, the IPFS commands will be same on macOS, Windows, and Linux. You will need to know how to navigate through your computers directories from within the CLI, if you are unsure how to do that, it is recommended to learn how before continuing with this guide.
+
+## Install IPFS using the CLI
+
+Installing IPFS using the CLI is simple, and there is already a great guide on how to do so [here](../install/command-line/).
+
+Once you have IPFS installed, you need to get your node up and running. If this is your first time using IPFS you will first need to initialize the configuration files.
+
+```
+ipfs init
+
+> initializing ipfs node at /Users/<user>/.ipfs
+> generating 2048-bit RSA keypair...done
+> peer identity: Qm...
+> to get started, enter:
+>
+>   ipfs cat /ipfs/QmYwAPJzv5CZsnA625s3Xf2nemtYgPpHdWEz79ojWnPbdG/readme
+```
+
+Your IPFS node is now initialized and you are ready to start the daemon to bring your node online.
+
+```
+ifps daemon
+
+> Initializing daemon...
+> go-ipfs version: 0.12.0
+> Repo version: 12
+> System version: arm64/darwin
+> ...
+```
+
+You're all set! Simply open up a new instance of the CLI and continue on with the guide!
+
+### Important Note
+
+Do not close the CLI that you used to initialize your daemon, only terminate the daemon when you want to take your IPFS node offline.
```

Comment:
Add notes like this into callout:

```suggestion
:::warning
Do not close the CLI that you used to initialize your daemon, only terminate the daemon when you want to take your IPFS node offline.
:::
```

This is a VuePress specific thing. You can't use this code on Lotus or Filecoin docs.